build/raise-version/Jenkinsfile

```diff
@@ -25,16 +25,16 @@ pipeline {
       steps {        
         script {
           currentBuild.description = "version: ${newVersion} branch: ${params.branch}"
-          if (!dryRun) {
-            mvn cmd: "-f build/new-jira-version/pom.xml initialize -DnewJiraVersion=${newVersion}"
-          }
```

Comment:
how will this be done now?

Reply:
I would say this was never executed, it was moved to the own pipeline: new-jira-version.
But Looks like this is also not used: https://jenkins.ivyteam.io/job/github-repo-manager_new-jira-version/

Reply:
A no sorry, I think in case of a dryRun it was used :eyes: 

It is strange because, per default you have to use prams.dryRun. But it looks like also dryRun can be used in the followup runs, just not in the first one ;)!